cfme/tests/ansible/test_embedded_ansible_tags.py

```diff
@@ -0,0 +1,130 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    test_requirements.ansible,
+    pytest.mark.uncollectif(lambda appliance: appliance.version < "5.9",
```

Comment:
**Optional**
Consider using the ignore_stream marker for this appliance version based uncollect.